src/syft/lib/__init__.py

```diff
@@ -88,12 +88,33 @@ def _load_lib(lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
         vendor_requirements=PACKAGE_SUPPORT
     ):
         update_ast = getattr(vendor_ast, "update_ast", None)
+        post_update_ast = getattr(vendor_ast, "post_update_ast", None)
         if update_ast is not None:
             global lib_ast
             update_ast(ast_or_client=lib_ast)
+            if post_update_ast is not None:
```

Comment:
MAYBE: Can we move lines 94 - 104 to another function, maybe called ```add_library``` and then call it lik this:
```
if update_ast is not None:
  global lib_ast
  add_library(lib_ast)
  for client in lib_ast.registered_clients.values():
    add_library(client)
```

Reply:
Great idea, iv gone a step further and done the same to the union regeneration code.